ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -95,10 +96,17 @@ export default defineComponent({
     }
   },
   setup (props) {
+    const { getLengthTrust } = useGetters<any>(['getLengthTrust'])
     const localState = reactive({
       feeType: props.setFeeType,
       registrationType: props.setRegistrationType,
-      registrationLength: props.setRegistrationLength,
+      registrationLength: computed((): RegistrationLengthI => {
+        if (localState.isValid !== true) {
+          props.setRegistrationLength.lifeYears = 0
```

Comment:
since props.setRegistrationLength can be null you will need to check it exists I think or use ? notation

Reply:
Hey sorry Steph, I think this will still err because it's not always set in our app. I think changing the prop in the child component is also a bad practice because it breaks the sync with the parent value. Do you think it would be better to put this validation in the length trust component and updating the life years there instead?

Reply:
This change was made because Scott wanted to show "not valid" versus unset.... happy to discuss

Reply:
ah so you probably need the actual life years value somewhere else I guess. Okay well that's fine, but this will still throw errors in the app where props.setRegistrationLength is null won't it? Amendment + Discharge?

Reply:
added a check for null in there.